docs/administration/deployment/deploying-locally/index.md

```diff
@@ -53,36 +58,50 @@ $ source ~/.virtualenvs/reana/bin/activate
 (reana) $ pip install git+git://github.com/reanahub/reana.git#egg=reana
 ```
 
-Fork and clone all REANA source code repositories:
+In addition, for **conda** users, set `VIRTUAL_ENV` variable:
+
+```console
+(reana) $ export VIRTUAL_ENV=/../../miniconda3/envs/reana/bin/python
+```
+
+**2.** Fork and clone all REANA source code repositories:
 
 ```console
 (reana) $ reana-dev git-fork --help
 (reana) $ eval "$(reana-dev git-fork -c ALL)"
 (reana) $ reana-dev git-clone -c ALL -u johndoe
 ```
 
-Create a new REANA cluster:
+For **MacOS**, add `-b open` to `git-fork` command, example:
+
+```console
+(reana) $ reana-dev git-fork -c ALL -b open
+```
+
```

Comment:
I'm not sure if we should include this or for something more generic like "You can customize the browser by adding the `-b` flag, e.g. macOS users can run ..."

Also, this command should be wrapped in the `eval` to open the browser directly, as it's above.